java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -3,228 +3,85 @@
         return props;
     }
 
-    public String getBootstrapServers() {
-        return bootstrapServers;
-    }
-
     public String getTopic() {
         return topic;
     }
 
-    public String getGroupId() {
-        return groupId;
-    }
-
-    public String getAutoOffsetReset() {
-        return autoOffsetReset;
-    }
-
     public String getEnableAutoCommit() {
         return enableAutoCommit;
     }
 
-    public String getClientRack() {
-        return clientRack;
-    }
-
     public Long getMessageCount() {
         return messageCount;
     }
 
-    public String getSslTruststoreCertificates() {
-        return sslTruststoreCertificates;
-    }
-
-    public String getSslKeystoreKey() {
-        return sslKeystoreKey;
-    }
-
-    public String getSslKeystoreCertificateChain() {
-        return sslKeystoreCertificateChain;
-    }
-
-    public String getOauthClientId() {
-        return oauthClientId;
-    }
-
-    public String getOauthClientSecret() {
-        return oauthClientSecret;
+    public TracingSystem getTracingSystem() {
+        return tracingSystem;
     }
 
-    public String getOauthAccessToken() {
-        return oauthAccessToken;
-    }
-
-    public String getOauthRefreshToken() {
-        return oauthRefreshToken;
-    }
-
-    public String getOauthTokenEndpointUri() {
-        return oauthTokenEndpointUri;
-    }
-
-    public String getAdditionalConfig() {
-        return additionalConfig;
-    }
-
-    public TracingSystem getTracingSystem() { return tracingSystem; }
-
     @Override
     public String toString() {
         return "KafkaConsumerConfig{" +
-            "bootstrapServers='" + bootstrapServers + '\'' +
-            ", topic='" + topic + '\'' +
-            ", groupId='" + groupId + '\'' +
-            ", autoOffsetReset='" + autoOffsetReset + '\'' +
-            ", enableAutoCommit='" + enableAutoCommit + '\'' +
-            ", clientRack='" + clientRack + '\'' +
-            ", messageCount=" + messageCount +
-            ", sslTruststoreCertificates='" + sslTruststoreCertificates + '\'' +
-            ", sslKeystoreKey='" + sslKeystoreKey + '\'' +
-            ", sslKeystoreCertificateChain='" + sslKeystoreCertificateChain + '\'' +
-            ", oauthClientId='" + oauthClientId + '\'' +
-            ", oauthClientSecret='" + oauthClientSecret + '\'' +
-            ", oauthAccessToken='" + oauthAccessToken + '\'' +
-            ", oauthRefreshToken='" + oauthRefreshToken + '\'' +
-            ", oauthTokenEndpointUri='" + oauthTokenEndpointUri + '\'' +
-            ", additionalConfig='" + additionalConfig + '\'' +
-            ", tracingSystem='" + tracingSystem + '\'' +
-            '}';
+                "topic='" + topic + '\'' +
+                ", enableAutoCommit='" + enableAutoCommit + '\'' +
+                ", messageCount=" + messageCount +
+                ", tracingSystem='" + tracingSystem + '\'' +
+                kafkaConfigOptionsToString() +
```

Comment:
Is this needed? The Kafka client prints its config at startup. If you want to keep this, you should check the formatting to fit into the other fields.

Reply:
Not needed now as we have ` kafkaConfigOptionsToString()`. Removed